README.md

```diff
@@ -22,6 +22,15 @@ Read the full Parse Server guide here: https://github.com/ParsePlatform/parse-se
 * You now have a database named "dev" that contains your Parse data
 * Install ngrok and you can test with devices
 
+### Helpful Scripts:
```

Comment:
> Thanks @mtrezza! Should I mention that `npm run watch` should only be used locally?

I think from the getting started guide of Parse Server it seems obvious that these commands are for development and not to launch parse server. But you could add a line like "These scripts can help you to develop your app for Parse Server:"